update/delta/delta_1.3.6_add_vl_for_SIA_export.sql

```diff
@@ -0,0 +1,237 @@
+/*
+This adds corresponding sia405 values for interlis import/export.
+*/
+
+-- Add columns
+
+ALTER TABLE qwat_vl.value_list_base ADD COLUMN sia405code TEXT;
+
+
+-- Populate columns
+
+-- TODO : Bedding material mapping complete [X] to check [ ]
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'aucun';
+UPDATE qwat_vl.bedding SET sia405code = 'Beton' WHERE value_fr = 'béton';
+UPDATE qwat_vl.bedding SET sia405code = 'Fliess' WHERE value_fr = 'géotextile';
+UPDATE qwat_vl.bedding SET sia405code = 'Betonkies' WHERE value_fr = 'gravier';
+UPDATE qwat_vl.bedding SET sia405code = 'Sand' WHERE value_fr = 'sable';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'terre';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cistern SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cover_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- Hydrant material : mapping complete [X] to check [ ]
+UPDATE qwat_vl.hydrant_material SET sia405code = 'Metall' WHERE short_fr LIKE 'F %';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_inf SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_sup SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_output SET sia405code = 'unbekannt' WHERE ???;
```

Comment:
Here I think I find (hidden) information whether it is einarmig, zweiarmig, dreiarmig: 2x55_1x75 or 3x75 - maybe we have to combine hydrant_model_* and hydrant_output for the Hydrant.Art matching in SIA405 water.